packages/vite/src/node/plugins/resolve.ts

```diff
@@ -375,12 +376,12 @@ export function tryNodeResolve(
     path.isAbsolute(importer) &&
     fs.existsSync(cleanUrl(importer))
   ) {
-    basedir = path.dirname(importer)
+    basedir = fs.realpathSync.native(path.dirname(importer))
```

Comment:
I'm not sure why we need `fs.realpathSync.native` for this. Perhaps a comment could help explain it.

Reply:
I don't remember why I added it xD

Removing for now..